EIPS/eip-867.md

```diff
@@ -0,0 +1,190 @@
+
+If accepted, an ERP could easily compile the block at while the changes are to take place, and the source of the actions (which would be the output of the script, in a standardized object output). These can be bundled with the client for seamless execution.
+
+
+## ERP Approval Process
+Approval of any ERP will require a mandatory minimum of 30-days between the time the ERP is submitted (or substantially modified) and adoption by clients, and ERPs affecting more funds or more addresses may prompt a considerably longer review period.  In practice, ERPs will often require more time because the will likely be bundled into other client upgrades that are on their own schedule.  However, the 30-day mandatory review period  guarantees a period for comment and auditing and prevents any last minute inclusions from being rushed into an upcoming upgrade.  
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that have a demonstrable history within the Ethereum community that are unaffected by the issue but may serve as relatively impartial adjudicators
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to this meta-EIP.  Community members, in general, are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
```

Comment:
I still maintain that "positive, explicit, and broad support" from the community should be required.  The restriction of the heard parties in this draft is the main reason that I believe this process will become "trusted operates at the helm", which the Ethereum philosophy tries to remove.

Reply:
I will consider this.  I do have some concerns about when it is appropriate to consider subjective feedback that does not address the facts of the claim, or if failure to get "broad" support from the entire community (due to apathy) should be a blocker.  If an ERP is submitted that provides conclusive evidence that they are the owners of the funds (e.g. consider a SafeSplitter example where funds were sent to the 0x0 address), it's not clear to me why broad community support is needed.  The broad support mentioned above it about broad support for factual claims from people that can reasonably be considered impartial and technically competent.  e.g. "is the signed message attached to this ERP from the address in question?".

Reply:
The broad support from the community is required for any hard-fork to become effective. If a hard-fork is a purely technical change, almost no communication is need (see shelling point: https://en.wikipedia.org/wiki/Focal_point_(game_theory) ), however, if there is a chance that someone will disagree, the Schelling point vanishes and amount of communication required greatly increases. Because people are not so sure anymore that everyone else will agree, so they start checking.

Therefore, I think adding recoveries into regular technical hard-forks greatly increases their cost (because no, you cannot just sneak them in). They need to be considered on their own merit, but then you go not get the uncontroversial change to "shield" you from the negative opinions.

Reply:
Some Ethereum users have a belief that the original owner should not get the funds in total.  The process needs to hear them too.  Otherwise, the process is not hearing them, but still making decisions about their network operates.

Reply:
@pirapira   
>Some Ethereum users have a belief that the original owner should not get the funds in total. 

Is this in reference to the proposals for things like a percentage "penalty", etc?

Reply:
> Some Ethereum users have a belief that the original owner should not get the funds in total. 

It's a valid point. Would this be applied on a case by case basis, though?  Would some addresses in the SafeSplitter case be given, say 80%, and others 10%?  This is the sort of this I'd like to avoid as I think it will result in special treatment as it becomes entirely subjective.

It seemed more reasonable to me to take a position like "if you can prove you own the funds AND demonstrate that there is no other person that could reasonably claim ownership" then the only downside to returning the funds is the effort required to include the change in the next hard fork.  This EIP attempts to minimize that effort.

Reply:
or, some people have suggested a flat percentage as a "penalty".... not advocating for or against, but something like that might a) help sway people who don't want just a "free pass" to screw up; and b) make it more objective

Reply:
@ryoungblom 
> Is this in reference to the proposals for things like a percentage "penalty", etc?

Yes, for example, [Evan Van Ness](http://www.weekinethereum.com/post/167279242888/november-8-2017) expressed this publicly:
> I’m not an Immutability Maximalist but any fork should include a haircut to protect against moral hazard.

Also, I said in [the ether recovery channel](https://gitter.im/ethereum/ether-recovery)
> I believe reckless users should be punished although I can tolerate nicer treatments for a while. (Nov 15 2017 11:02)